code/modules/vehicles/ridden/wheelchair.dm

```diff
@@ -59,7 +91,7 @@
 		R.color = src.color
 		qdel(src)
 
-/obj/vehicle/ridden/wheelchair/OnMouseDropLegacy(over_object, src_location, over_location)
```

Comment:
Is this intended to overwrite the Legacy mouse drop?
I know silicons recently added the legacy exact

Reply:
Oh, I saw that there was OnMouseDropLegacy and OnMouseDrop and thought OnMouseDrop was the more recent proc. OnMouseDrop in game works pretty much the same from what Ive tested

Reply:
Legacy is depricated use non legacy